src/test.js

```diff
@@ -647,8 +658,11 @@ export function only( testName, callback ) {
 		return;
 	}
 
-	config.queue.length = 0;
-	focused = true;
+	// Only empty the queue in case this test is not part of a skipped module
+	if ( !config.currentModule.skip ) {
```

Comment:
Per the above, let's remove this. We should always "focus" the test, even if it will wind up being skipped.